ui/pages/confirmations/components/confirm/info/personal-sign/personal-sign.tsx

```diff
@@ -85,6 +102,13 @@ const PersonalSignInfo: React.FC = () => {
           </ConfirmInfoAlertRow>
         )}
       </ConfirmInfoSection>
+      {isSnapId && (
+        <SnapMetadataModal
```

Comment:
It would probably be more ideal if this wasn't tied to `personal-sign`. Perhaps we can alter `ConfirmInfoRowUrl` to support Snaps so it works across all screens?

Reply:
yeah sure, makes sense.